microsoft-edge/devtools-guide-chromium/css/reference.md

```diff
@@ -306,114 +306,31 @@ To add a new style rule:
 
    ![Add a new style rule.](../media/css-elements-styles-style-new.msft.png)
 
-<!-- are we recommending continuing from the above then doing the following?  or does the following give more detail on how or where to do the above? -->
-
 
 ### Select a stylesheet to add a rule to
 
-When [adding a new style rule](#add-a-style-rule), to select a stylesheet to add a rule to:
+By default, when adding a style rule, DevTools creates a new stylesheet named `inspector-stylesheet` in the document and then adds the new style rule in this stylesheet.
 
-*  Click and hold **New Style Rule** (![New Style Rule.](../media/new-style-rule-icon.msft.png)) to select a stylesheet to add the style rule to.
+To instead add the rule in an existing stylesheet:
 
-![Selecting a stylesheet.](../media/css-elements-styles-style-new-select-existing.msft.png)
+*  Click and hold **New Style Rule** (![New Style Rule.](../media/new-style-rule-icon.msft.png)) and then select a stylesheet from the list to add the style rule to.
 
-<!-- is the above an alternative to the below, or sequential? clarify flow/relation -->
+![Selecting a stylesheet.](../media/css-elements-styles-style-new-select-existing.msft.png)
 
 
 ### Add a style rule to a specific location
 
-To add a style rule to a specific location in the **Styles** panel:<!--is this opposed to some above approach?-->
+By default, adding a style rule by clicking on **New Style Rule** inserts the rule beneath the **element.style** rule.
 
-1. Hover on the style rule that is directly above where you want to add your new style rule.
+To add a style rule in a specific location of the **Styles** panel instead:
 
-1. [Reveal the **More Actions** toolbar](#reveal-the-more-actions-toolbar).
+1. Hover on the style rule that is directly above where you want to add your new style rule.
 
-1. Select **Insert Style Rule Below** (![Insert Style Rule Below icon.](../media/new-style-rule-icon.msft.png)).
+1. Click **Insert Style Rule Below** (![Insert Style Rule Below icon.](../media/new-style-rule-icon.msft.png)).
 
 ![Insert Style Rule Below.](../media/css-elements-styles-insert-style-rule-below.msft.png)
 
 
```

Comment:
maybe add a line below screenshot:
The new style rule is added in the same stylesheet as the rule above it.

Reply:
See my previous comment. This isn't always correct. It looks to me like if all rules in the Styles panel are already in the same stylesheet, then that's how it works. Otherwise, a different (seemingly random) stylesheet is used to create the new rule.